src/bin/console/console.cpp

```diff
@@ -588,6 +597,29 @@ int Console::visualize(const std::string& input) {
   return ReturnCode::Ok;
 }
 
+int Console::change_runtime_setting(const std::string& input) {
```

Comment:
Once we do have multiple methods, we could move them into their own methods or have some generalized way of doing that. This could also relate to #63

Reply:
Correct